curriculum/challenges/english/25-front-end-development/lab-lunch-picker-program/66db529d37ad966480ebb633.md

```diff
@@ -49,20 +60,34 @@ You should define a function `addLunchToEnd`.
 assert.isFunction(addLunchToEnd);
 ```
 
-The function `addLunchToEnd` should take a single argument.
+The function `addLunchToEnd` should have two parameters.
 
 ```js
-assert.lengthOf(addLunchToEnd, 1);
+assert.lengthOf(addLunchToEnd, 2);
 ```
 
-`addLunchToEnd("Tacos")` should return the string `"Tacos added to the end of the lunch menu."`.
+`addLunchToEnd(lunches, "Tacos")` should log the string `"Tacos added to the end of the lunch menu."` to the console.
 
 ```js
-assert.strictEqual(addLunchToEnd("Tacos"), "Tacos added to the end of the lunch menu.");
+const tempArr = [];
+const temp = console.log;
+const testLunches = []
+try {
+  console.log = obj => tempArr.push(obj);
+  addLunchToEnd(testLunches, "Tacos");
+  assert.strictEqual(tempArr[0], "Tacos added to the end of the lunch menu.");
+  addLunchToEnd(testLunches, "Pizza");
+  assert.strictEqual(tempArr[1], "Pizza added to the end of the lunch menu.");
+} finally {
+  console.log = temp;
+}
+```
+
+`addLunchToEnd(["Pizza", "Tacos"], "Burger")` should return `["Pizza", "Tacos", "Burger"]`.
 
-// prevent hardcoding
-assert.strictEqual(addLunchToEnd("Pizza"), "Pizza added to the end of the lunch menu.");
-assert.strictEqual(addLunchToEnd("Burger"), "Burger added to the end of the lunch menu.");
+```js
+assert.deepStrictEqual(addLunchToEnd(["Pizza", "Tacos"], "Burger"), ["Pizza", "Tacos", "Burger"]);
+assert.deepStrictEqual(addLunchToEnd(["Fries"], "Tacos"), ["Fries", "Tacos"]);
```

Comment:
```suggestion
const temp = console.log;
console.log = () => {}
try {
  assert.deepStrictEqual(addLunchToEnd(["Pizza", "Tacos"], "Burger"), ["Pizza", "Tacos", "Burger"]);
  assert.deepStrictEqual(addLunchToEnd(["Fries"], "Tacos"), ["Fries", "Tacos"]);
} finally {
  console.log = temp;
}
```
We need to override `console.log` here otherwise campers can see these strings there and it can be confusing.